src/mcbopomofo-addon.conf.in.in

```diff
@@ -1,5 +1,6 @@
 [Addon]
 Name=McBopomofo
+Comment=McBopomofo Wrapper For Fcitx
```

Comment:
Do other addons use the same term? Just wondering whether we can just call this `McBopomofo for fcitx5`.